[2024-05-04T15:38:44.365+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-04T15:38:44.380+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_space.check_space manual__2024-05-04T15:38:44.164023+00:00 [queued]>
[2024-05-04T15:38:44.385+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_space.check_space manual__2024-05-04T15:38:44.164023+00:00 [queued]>
[2024-05-04T15:38:44.385+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-05-04T15:38:44.393+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): check_space> on 2024-05-04 15:38:44.164023+00:00
[2024-05-04T15:38:44.397+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_space', 'check_space', 'manual__2024-05-04T15:38:44.164023+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/dag_mysql.py', '--cfg-path', '/tmp/tmpqjrde_9j']
[2024-05-04T15:38:44.398+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=987) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-04T15:38:44.399+0000] {standard_task_runner.py:91} INFO - Job 85: Subtask check_space
[2024-05-04T15:38:44.399+0000] {standard_task_runner.py:63} INFO - Started process 988 to run task
[2024-05-04T15:38:44.453+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_space.check_space manual__2024-05-04T15:38:44.164023+00:00 [running]> on host f85b7a0b58b5
[2024-05-04T15:38:44.495+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='bses' AIRFLOW_CTX_DAG_ID='dag_space' AIRFLOW_CTX_TASK_ID='check_space' AIRFLOW_CTX_EXECUTION_DATE='2024-05-04T15:38:44.164023+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-04T15:38:44.164023+00:00'
[2024-05-04T15:38:44.496+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-04T15:38:44.503+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-04T15:38:44.504+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'pwd']
[2024-05-04T15:38:44.508+0000] {subprocess.py:86} INFO - Output:
[2024-05-04T15:38:44.509+0000] {subprocess.py:93} INFO - /tmp/***tmpvdqnik1n
[2024-05-04T15:38:44.509+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-04T15:38:44.509+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-04T15:38:44.523+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_space, task_id=check_space, execution_date=20240504T153844, start_date=20240504T153844, end_date=20240504T153844
[2024-05-04T15:38:44.555+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-04T15:38:44.566+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-04T15:38:44.567+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
